Makefile

```diff
@@ -7,6 +7,8 @@ VERSION_REPLACES ?= 0.79.6
 
 DEPLOY_DIR ?= manifests
 
+MAX_GO_VERSION=1.18
```

Comment:
needs to be 1.19

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/pull/2228#issuecomment-2781299233
depends on us, atm it is v1.18, question is if we want to just enforce, or also update 
asking because it is very old branch

Reply:
ah missed that comment.

IMO we should align to the enforced version, it's never too late :) 
If it breaks something we should know by our U/S (and other) CI lanes.

Reply:
updated